libs/jwst-codec/src/doc/document.rs

```diff
@@ -312,49 +312,44 @@ mod tests {
     }
 
     #[test]
+    #[cfg_attr(any(miri, loom), ignore)]
```

Comment:
cannot pass loom test?

Reply:
loom always reports "Model exceeded maximum number of branches" at `array.insert(idx, sub_arr)`, will look on it later on